File: Easy/Chuck Norris/Java/ChuckNorris.java
import java.util.*;
import java.io.*;

class Solution {

    public static void main(String args[]) throws UnsupportedEncodingException {
        Scanner in = new Scanner(System.in);
        String message = in.nextLine();

        byte[] bArray = message.getBytes("US-ASCII");

        // Example: "C" >> "67"
        String binary = "";
        for (int i = 0; i < bArray.length; i++)
            binary += String.format("%7s", (Integer.toBinaryString(bArray[i]))).replace(' ', '0');

        char[] chars = binary.toCharArray();

        // It means the beginning of text
        int lastPosition    = 2;

        // ASCII >> 49 = 1; 48 = 0
        for (int i = 0; i < chars.length; i++)
        {
            if (chars[i] == '1')
                System.out.print(lastPosition == 1 ? "0" : lastPosition == 0 ? " 0 0" : "0 0");
            else
                System.out.print(lastPosition == 0 ? "0" : lastPosition == 1 ? " 00 0" : "00 0");

            lastPosition = (chars[i] == '1' ? 1 : 0); // new last bit; // new last bit
        }
    }
}

File: Easy/Chuck Norris/Python 3/Chuck Norris.py
message = input()
binary_message = ' '.join(format(ord(every_character), 'b') for every_character in message).zfill(7).replace(" ", "")
print("binary_message = " + binary_message, file=sys.stderr)

binary_message = ' '.join(format(ord(every_character), 'b') for every_character in message)

last_position = 2 # It means the beginning of text
encoded_message = ""

for i in range(len(binary_message)):
    if (int(binary_message[i]) == 1):
        encoded_message += "0" if last_position == 1 else " 0 0" if last_position == 0 else "0 0"
    else:
        encoded_message += "0" if last_position == 0 else " 00 0" if last_position == 1 else "00 0"

    last_position = 1 if int(binary_message[i]) == 1 else 0 # new last bit

print(encoded_message);